### DISCUSSION TITLE
Filter (order?) to largest repos

### DISCUSSION THREAD

**Discussion by gsnedders** [community/community#13821] [Other Feature Feedback, Questions, & Ideas]
One thing that comes up frequently when working on major libraries or other software development platforms is questions like "how widely used is [X]" or "how is [X] used in the wild".

While code search can answer those questions, it's often more interesting to know "what are the biggest repos with [X]" (stars? forks?) and look at them.

**Comment by garthhumphreys** [comment]
I agree, it would be wonderful to filter the results by Stars
